neuralop/models/fnogno.py

```diff
@@ -13,6 +13,98 @@
+        fno_use_mlp : bool, optional
+            Whether to use an MLP layer after each FNO block. Defaults to False.
+        fno_mlp_dropout : float, optional
+            dropout parameter of above MLP. Defaults to 0.
+        fno_mlp_expansion : float, optional
+            expansion parameter of above MLP. Defaults to 0.5.
+        fno_non_linearity : nn.Module, optional
+            nonlinear activation function between each FNO layer. Defaults to F.gelu.
+        fno_stabilizer : nn.Module, optional
+            By default None, otherwise tanh is used before FFT in the FNO block. 
+        fno_norm : nn.Module, optional
+            normalization layer to use. Defaults to None.
+        fno_ada_in_features : _type_, optional
+            if an adaptive mesh is used, number of channels of its positional embedding. Defaults to None.
+        fno_ada_in_dim : int, optional
+            dimensions of above FNO adaptive mesh. Defaults to 1.
+        fno_preactivation : bool, optional
+            whether to use Resnet-style preactivation. Defaults to False.
+        fno_skip : str, optional
+            type of skip connection to use. Defaults to 'linear'.
+        fno_mlp_skip : str, optional
+            type of mlp skip connection to use, by default 'soft-gating'
+        fno_separable : bool, optional
+            if True, use a depthwise separable spectral convolution. Defaults to False.
+        fno_factorization : str {'tucker', 'tt', 'cp'}, optional
+            Tensor factorization of the parameters weight to use. Defaults to None.
+        fno_rank : float, optional
+            Rank of the tensor factorization of the Fourier weights. Defaults to 1.0.
+        fno_joint_factorization : bool, optional
+            Whether all the Fourier layers should be parameterized by a single tensor (vs one per layer). Defaults to False.
+        fno_fixed_rank_modes : bool, optional
+            Modes to not factorize. Defaults to False.
+        fno_implementation : str {'factorized', 'reconstructed'}, optional
+            If factorization is not None, forward mode to use::
+            * `reconstructed` : the full weight tensor is reconstructed from the factorization and used for the forward pass
+            * `factorized` : the input is directly contracted with the factors of the decomposition
+        fno_decomposition_kwargs : dict, optional
+            Optionaly additional parameters to pass to the tensor decomposition. Defaults to dict()
+        fno_domain_padding : float, optional
+            If not None, percentage of padding to use. Defaults to None.
+        fno_domain_padding_mode : str {'symmetric', 'one-sided'}, optional
+            How to perform domain padding. Defaults to 'one-sided'.
+        fno_fft_norm : str, optional
+            normalization param of torch.fft to use in FNO. Defaults to 'forward'
+        fno_SpectralConv : nn.Module, optional
```

Comment:
I like to have directly default is ... instead of optional since having a default value makes it optional anyway.